packages/shared/src/utils/patientCertificates/printComponents/PatientDetails.jsx

```diff
@@ -18,13 +18,13 @@ const PATIENT_FIELDS = {
   ],
 };
 
-export const PatientDetails = ({ patient, getLocalisation }) => {
+export const PatientDetails = ({ patient, getLocalisation, getSetting }) => {
```

Comment:
Hmmm it's a bit awkward having to pass this prop all the way down. But the `useSettings()` context hook doesn't exist in the `@tamanu/shared` package hey?

It looks like we do something custom with the language context, but since the settings rely on redux it might be weird to try and migrate it to shared. Happy with this then :+1: